src/System.Net.Security/src/System/Net/SecureProtocols/_SecuritySafeHandles.Windows.cs

```diff
@@ -402,7 +291,7 @@ internal sealed class SafeCredentialReference : CriticalHandleMinusOneIsInvalid
         //
         // Static cache will return the target handle if found the reference in the table.
         //
-        internal SafeFreeCredentials _Target;
+        internal SafeFreeCredentials _target;
```

Comment:
We can take this as a seperate discussion/exercise. We tried to compare & use the latest bit, but if  fixes got left out, we might need to get that in. CC: @vijaykota